README.md

```diff
@@ -29,6 +29,20 @@ Run the following command at the root of your documentation (where mint.json is)
 pnpm dev
 ```
 
+### API docs
+
+api docs is done based on a openapi.yaml definition. There is a few options on how to do this see
+[api docs with mintlify](https://mintlify.com/docs/api-playground/openapi/setup).
+
+In this repo we use the MDX way, that means we leverage the scraper tool.
+
+```
+npx @mintlify/scraping@latest openapi-file ./api-ref/openapi.yaml -o api-ref
+```
+
+Then all generated MDX can be added additional information if needed. Tags in the openapi.yaml definition
```

Comment:
the first sentence has some missing word or grammar issue, I don't get it at least 😬